### PR TITLE
Clear nodes cache

### PR DESCRIPTION
**Pull request by ebattat**
This PR solve Insufficient memory.
Running clear cache method in initialize workload.

### CODE REVIEW

File: .github/workflows/Weekly_Func_Env_IPI_CI.yml [hunk 1]
```diff
@@ -61,7 +62,7 @@ jobs:
         # green when failed because retry
         continue-on-error: true
         env:
-          INSTALL_OCP_VERSION: "candidate-4.10"
```
Comment (RobertKrawitz) [inline .github/workflows/Weekly_Func_Env_IPI_CI.yml:64]: Why are these changes here?

File: benchmark_runner/common/oc/oc.py [hunk 2]
```diff
@@ -148,6 +148,12 @@ def get_worker_nodes(self):
         """
         return self.run(r""" oc get nodes -l node-role.kubernetes.io/worker= -o jsonpath="{range .items[*]}{.metadata.name}{'\n'}{end}" """)
 
+    def clear_nodes_cache(self):
+        """
+        This method clear nodes cache
```
Comment (RobertKrawitz) [inline benchmark_runner/common/oc/oc.py:153]: "clears the node's buffer cache"

**Review by RobertKrawitz — COMMENTED** [review]
Nits.

**Comment by ebattat** [comment]
Closed
